medicinskij-academicheskij-jurnal.csl

```diff
@@ -0,0 +1,190 @@
+          <text term="edition" form="short"/>
+        </group>
+      </if>
+      <else>
+        <text variable="edition"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="type-number">
+    <choose>
+      <if type="patent">
+        <text term="letter" form="short"/>
+        <text variable="number" prefix=" " suffix=" USA"/>
+      </if>
+      <else>
+        <group delimiter=" ">
+          <text variable="genre"/>
+          <text variable="number"/>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <macro name="container">
+    <choose>
+      <if type="article-journal">
+        <text variable="container-title" form="short"/>
+      </if>
+      <else>
+        <text variable="container-title"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="publisher">
+    <group delimiter=": ">
+      <text variable="publisher-place"/>
+      <text variable="publisher"/>
+    </group>
+  </macro>
+  <macro name="issued-year">
+    <date variable="issued">
+      <date-part name="year"/>
+    </date>
+  </macro>
+  <macro name="locators">
+    <group delimiter=", ">
+      <group delimiter=" ">
+        <text term="volume" form="short" text-case="sentence"/>
+        <text variable="volume"/>
+      </group>
+      <group delimiter=" ">
+        <text term="issue" form="symbol"/>
+        <text variable="issue"/>
+      </group>
+    </group>
+  </macro>
+  <macro name="page">
+    <group delimiter=" ">
+      <label variable="page" form="short" text-case="sentence" plural="never"/>
+      <text variable="page"/>
+    </group>
+    <group delimiter=" ">
+      <text variable="number-of-pages"/>
+      <text term="page" form="short"/>
+    </group>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if type="webpage">
+        <choose>
+          <if variable="URL">
+            <text variable="URL" prefix="URL: "/>
+            <group prefix=" (" suffix=")" delimiter=": ">
+              <text term="accessed"/>
+              <date variable="accessed">
+                <date-part name="day" suffix="." form="numeric-leading-zeros"/>
+                <date-part name="month" suffix="." form="numeric-leading-zeros"/>
+                <date-part name="year"/>
+              </date>
+            </group>
+          </if>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <citation collapse="citation-number">
+    <sort>
+      <key variable="citation-number"/>
+    </sort>
+    <layout prefix="[" suffix="]" delimiter=",">
+      <text variable="citation-number"/>
+      <group prefix=", " delimiter=" ">
+        <label variable="locator" form="short" plural="never"/>
+        <text variable="locator"/>
+      </group>
+    </layout>
+  </citation>
+  <bibliography et-al-min="4" et-al-use-first="1" entry-spacing="0" second-field-align="flush">
+    <layout suffix=".">
+      <text variable="citation-number" suffix="."/>
+      <group delimiter=".вЂ“ ">
```

Comment:
@adam3smith, I think this is an encoding issue (same with `<summary/>` as discussed above, and with the `<locale/>` terms). Looks to be WINDOWS-1251 instead of UTF-8.

@ValeryMukhin, do you use a text editor on your computer for editing this style. I'm wondering if you could maybe change the text encoding setting to "UTF-8" so that the Russian text will show up correctly here.